boa_engine/src/error.rs

```diff
@@ -673,6 +755,11 @@ impl JsNativeError {
     /// assert!(error_obj.borrow().is_error());
     /// assert_eq!(error_obj.get("message", context).unwrap(), "error!".into())
     /// ```
+    ///
+    /// # Panics
+    ///
+    /// If the converting a [`JsNativeErrorKind::RuntimeLimit`] to an opaque object.
```

Comment:
This also happens with `NoInstructionsRemain`, right?

Reply:
When we enable `"fuzz"` feature we use with the fuzz targets and this shouldn't be reached, that why it throws an `unreachable!()`, unlike `RuntimeLimit` which uses a `panic!`.